ibis/bigquery/client.py

```diff
@@ -397,50 +381,3 @@ def get_schema(self, name, database=None):
     @property
     def version(self):
         return parse_version(bq.__version__)
-
-
-_DTYPE_TO_IBIS_TYPE = {
-    'INT64': dt.int64,
-    'FLOAT64': dt.double,
-    'BOOL': dt.boolean,
-    'STRING': dt.string,
-    'DATE': dt.date,
-    # FIXME: enforce no tz info
-    'DATETIME': dt.timestamp,
-    'TIME': dt.time,
-    'TIMESTAMP': dt.timestamp,
-    'BYTES': dt.binary,
-}
-
-
-_LEGACY_TO_STANDARD = {
-    'INTEGER': 'INT64',
-    'FLOAT': 'FLOAT64',
-    'BOOLEAN': 'BOOL',
-}
-
-
-def _discover_type(field):
-    typ = field.field_type
-    if typ == 'RECORD':
-        fields = field.fields
-        assert fields
-        names = [el.name for el in fields]
-        ibis_types = [_discover_type(el) for el in fields]
-        ibis_type = dt.Struct(names, ibis_types)
```

Comment:
Note: `to_dataframe()` in the `google-cloud-bigquery` library just uses `object` as the datatype for record and array columns. (I think because Pandas doesn't have these datatypes) Will that cause a problem for Ibis? Is there actually a Pandas Struct and Array type now that we should be using in that library?

Reply:
There's no struct/array type in pandas. This won't affect ibis, since we already assume this to be true.

Reply:
👍 

Would it make sense for `google-cloud-bigquery` to add a `to_arrow()` method like `to_dataframe()` so that these types have better support here?